### ISSUE TITLE
Freeze of wordpress pod after the installation wizard

### ISSUE THREAD

**Issue by medelgover** [bitnami/charts#6247]
**Which chart**:
The name (and version) of the affected chart
chart: bitnami/wordpress

**Describe the bug**
After fresh installation of the chart - MariaDB starts properly and works, but WordPress fails during the **installation wizard**. It visualizes the wizard and collects data, however, once the GUI button clicked "install" - it fails, and the pod restarts. 

in logs at the same time: 
10.10.0.143 - - [29/Apr/2021:04:49:13 +0000] "GET /wp-admin/install.php HTTP/1.1" 200 2665
**10.10.0.143 - - [29/Apr/2021:04:49:23 +0000] "GET /wp-admin/install.php HTTP/1.1" 500 2628**
10.10.0.143 - - [29/Apr/2021:04:49:18 +0000] "GET /wp-admin/install.php HTTP/1.1" 200 2171
10.10.0.143 - - [29/Apr/2021:04:49:33 +0000] "GET /wp-admin/install.php HTTP/1.1" 200 518

The same situation with option: 
wordpressSkipInstall: false

Inside the logs there is 500 - and after this pod of WordPress is crashing 

**To Reproduce**

1. Fresh installation
2. To use the wizard via GUI - or - to user option: wordpressSkipInstall: false

**Expected behavior**

**Version of Helm and Kubernetes**:

- Output of `helm version`:

```
~$ helm version
version.BuildInfo{Version:"v3.4.1", GitCommit:"c4e74854886b2efe3321e185578e6db9be0a6e29", GitTreeState:"clean", GoVersion:"go1.14.11"}

```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.4", GitCommit:"d360454c9bcd1634cf4cc52d1867af5491dc9c5f", GitTreeState:"clean", BuildDate:"2020-11-11T13:17:17Z", GoVersion:"go1.15.2", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"19", GitVersion:"v1.19.7", GitCommit:"1dd5338295409edcfff11505e7bb246f0d325d15", GitTreeState:"clean", BuildDate:"2021-01-13T13:15:20Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}

```

**Additional context**

**Comment by juan131** [comment]
Hi @medelgover 

I was unable to reproduce the issue. These are the steps I followed:

- Install MariaDB chart:

```console
$ helm install mariadb bitnami/mariadb \
  --set auth.database=bitnami_wordpress \ 
  --set auth.username=bn_wordpress \ 
  --set auth.password=wppassword
```

- Install WP chart (disabling probes to avoid WP to be restarted while the Wizard is passed) 

```console
$ helm install wordpress bitnami/wordpress \
  --set wordpressSkipInstall=true \
  --set mariadb.enabled=false \
  --set readinessProbe.enabled=false \
  --set livenessProbe.enabled=false \
  --set externalDatabase.user=bn_wordpress \
  --set externalDatabase.password=wppassword \
  --set externalDatabase.database=bitnami_wordpress \
  --set externalDatabase.host=mariadb.default.svc.cluster.local \
  --set externalDatabase.port=3306
```

- Wait for WP to be ready:

```console
$ kubectl logs -f $(kubectl get pod -l app.kubernetes.io/instance=wordpress -o jsonpath='{.items[0].metadata.name}')
(...)
wordpre INFO  ########################################################################
wordpre INFO   Installation parameters for wordpress:
wordpre INFO     Skip wizard: yes
wordpre INFO     Table Prefix: wp_
wordpre INFO   This installation requires no credentials.
wordpre INFO  ########################################################################
wordpre INFO
nami    INFO  wordpress successfully initialized
INFO  ==> Starting gosu...
$ kubectl port-forward svc/wordpress 8080:80
```

I could add the site settings, create my admin user and access the blog:

![Screenshot 2021-04-29 at 10 32 29](https://user-images.githubusercontent.com/6740773/116523243-522bc500-a8d6-11eb-9de9-609c9df6ffaf.png)
![Screenshot 2021-04-29 at 10 32 39](https://user-images.githubusercontent.com/6740773/116523250-535cf200-a8d6-11eb-98f6-8cf53e5fd24f.png)
![Screenshot 2021-04-29 at 10 32 55](https://user-images.githubusercontent.com/6740773/116523258-5657e280-a8d6-11eb-93da-60b7873cbb56.png)

**Comment by juan131** [comment]
Hi @medelgover 

Some comments regarding your values.yaml:

- You can skip these properties since you're using `wordpressSkipInstall=true` (therefore they'll be ignored):

```yaml
wordpress:
  wordpressBlogName: wordpress.lab.lcl
  wordpressUsername: admin
  wordpressPassword: "secure"
```

You can also skip this block since you're using `mariadb.enabled=true`:

```yaml
  externalDatabase:
    host: blog-mariadb
    user: bn_wordpress
    password: "password_wordpress"
    database: bitnami_wordpress
    port: 3306
```

--- 

Regarding your issue: is it possible that you're not installing the chart on a "clean" environment? I mean, with existing PVCs from previous installations of the chart? That's a common issue and we have documented how to workaround it, see:

- https://docs.bitnami.com/general/how-to/troubleshoot-helm-chart-issues/#persistence-volumes-pvs-retained-from-previous-releases

Could you please ensure you disable the readiness/liveness probes as I did in my example? You can enable them back once you pass the wizard.

```diff
wordpress:
+ readinessProbe:
+   enabled: false
+ livenessProbe:
+   enabled: false
```

**Comment by medelgover** [comment]
Hello @juan131 

First of all, thank you for your comments. 

Let me add a few details to make the situation for making it better to understand. 

1. Indeed - the installation is looking completed on chart:  
   Chart version: 10.10.3  
    WordPress image 5.7.1-debian-10-r0 
    Mariadb image: 10.5.9-debian-10-r28
   (by a manual update of the page after 5 or 6 minutes)
   - with **external** database 
   - with "internal" DB (which is part of WordPress). 
2. Answering your comment about "clean" environment - each test was true as a clean new deployment (PVC have been fully destroyed during each of the tests) 


Additionally I had: 
- The latest test with external DB showed the installation process was extremely slow (around 4-5 minutes)
- The only working url is: */wp-admin/install.php (which gives me the feeling that installation wasn't fully completed successfully) 

![Screenshot from 2021-05-04 22-49-52](https://user-images.githubusercontent.com/16061061/117061936-7438a380-ad2b-11eb-95b7-3e345f3e9d38.png)

The logs: 
```
10.10.0.143 - - [05/May/2021:14:00:48 +0000] "GET /wp-admin/install.php HTTP/1.1" 200 518
10.10.0.143 - - [05/May/2021:14:00:58 +0000] "GET /wp-admin/install.php HTTP/1.1" 200 518
10.10.0.143 - - [05/May/2021:14:01:08 +0000] "GET /wp-admin/install.php HTTP/1.1" 200 518
10.10.0.143 - - [05/May/2021:14:01:18 +0000] "GET /wp-admin/install.php HTTP/1.1" 200 518
10.10.0.143 - - [05/May/2021:14:01:28 +0000] "GET /wp-admin/install.php HTTP/1.1" 200 518
10.10.0.143 - - [05/May/2021:14:01:38 +0000] "GET /wp-admin/install.php HTTP/1.1" 200 518
10.10.0.143 - - [05/May/2021:14:01:48 +0000] "GET /wp-admin/install.php HTTP/1.1" 200 518
10.10.0.143 - - [05/May/2021:14:01:58 +0000] "GET /wp-admin/install.php HTTP/1.1" 200 518
10.10.0.143 - - [05/May/2021:14:02:08 +0000] "GET /wp-admin/install.php HTTP/1.1" 200 518
10.10.0.143 - - [05/May/2021:14:02:18 +0000] "GET /wp-admin/install.php HTTP/1.1" 200 518
10.10.0.143 - - [05/May/2021:14:02:28 +0000] "GET /wp-admin/install.php HTTP/1.1" 200 518
10.10.0.143 - - [05/May/2021:14:02:38 +0000] "GET /wp-admin/install.php HTTP/1.1" 200 518
10.10.0.143 - - [05/May/2021:14:02:48 +0000] "GET /wp-admin/install.php HTTP/1.1" 200 518
10.10.0.143 - - [05/May/2021:14:02:58 +0000] "GET /wp-admin/install.php HTTP/1.1" 200 518
10.10.0.143 - - [05/May/2021:14:03:08 +0000] "GET /wp-admin/install.php HTTP/1.1" 200 518
10.10.0.143 - - [05/May/2021:14:03:18 +0000] "GET /wp-admin/install.php HTTP/1.1" 200 518
10.10.0.143 - - [05/May/2021:14:03:28 +0000] "GET /wp-admin/install.php HTTP/1.1" 200 518
```

Current status:
```

NAME                            READY   STATUS    RESTARTS   AGE   IP               NODE        NOMINATED NODE   READINESS GATES
blog-mariadb-0                  1/1     Running   0          12m   10.233.106.47    k8scl2n06   <none>           <none>
blog-wordpress-9c9c5c44-zcs2q   **0/1**     Running   0          12m   10.233.127.108   k8scl2n04   <none>           <none>

```

**Comment by medelgover** [comment]
Hello @juan131 

After a huge amount of test - and comparison with minikube installation - I discovered that this behaviour is coming from ceph-rdb storage class. Once I  switched to cephfs storage class - it become workable. Could you, please, confirm that there is some dependencies on that? at the same time, the MariaDB chart is working fine with ceph-rdb (on the same cluster)

**Comment by juan131** [comment]
Hi @medelgover 

I am afraid I'm not familiar with that storageclass nor Ceph. Taking a look at:

- https://kubernetes.io/docs/concepts/storage/storage-classes/#ceph-rbd

It seems like you can choose the **fsType** when declaring the storageclass. What file system did you choose? It might be related with that (just guessing since I'm not an expert on this as a said)

**Comment by medelgover** [comment]
@juan131 
It was RDB storage class on top of ceph for MariaDB, and cephfs for WordPress. By using the same storageclass for WordPress - i got all situations above.

**Comment by marcosbc** [comment]
Hi, FYI we released WordPress chart 11.0.0 with many internal changes, could you try it and let us know if it continues failing?

**Comment by medelgover** [comment]
Colleagues, the new version works like a charm. Thank you for your help.
